net.sourceforge.liferea.json

```
@@ -0,0 +1,63 @@
+{
+    "app-id":"net.sourceforge.liferea",
+    "runtime":"org.gnome.Platform",
+    "runtime-version":"3.36",
```

Update runtime to 3.38
```suggestion
    "runtime-version":"3.38",
```